docs/docs/experimentation-analysis/experiment-time-series.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Experiment Time Series
+slug: /app/experiment-time-series
+---
+
+# Experiment Time Series
+
+Time series showing the cumulative results of your experiment though a given experiment update are available in-line in the experiment results.
```

Comment:
This sentence is a little confusing.

Suggestion: 

"The time series chart shows the cumulative results of your experiment. It's accessible directly from the experiment results view."

Reply:
Crazy inability to self-review, thanks.